website/content/partials/general_options_no_namespace.mdx

```diff
@@ -8,6 +8,10 @@
 - `-no-color`: Disables colored command output. Alternatively,
   `NOMAD_CLI_NO_COLOR` may be set.
 
+-`-force-color`: Forces colored command output. This can be used in cases where
+  the usual	terminal detection fails (CI systems mainly).	Alternatively,
```

Comment:
```suggestion
  the usual terminal detection fails (CI systems mainly).	Alternatively,
```

Reply:
Thanks that paragraph had another 3 issues like that as well, not sure how/why that hapenned :D